sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosCatalog.scala

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.spark
+
+import java.util
+// scalastyle:off underscore.import
+// scalastyle:on underscore.import
+import org.apache.spark.sql.catalyst.analysis.{NamespaceAlreadyExistsException, NoSuchNamespaceException}
+import org.apache.spark.sql.connector.catalog.{NamespaceChange, SupportsNamespaces}
+
+// scalastyle:off underscore.import
+
+class CosmosCatalog
```

Comment:
just for my curiosity, why do we need to add CosmosCatalog to both spark 3-1 and 3-2?

Reply:
Nevermind, just saw the movement of CosmosCatalog to all three modules now, thanks!